src/main.rs

```diff
@@ -28,11 +29,20 @@ const AGGREGATED: &str = "aggregated";
 const INTERVAL: std::time::Duration = std::time::Duration::from_millis(100);
 const MAX_DELAY_TIME: std::time::Duration = std::time::Duration::from_secs(120);
 
+// Introducing a simple cache for Receipts to find their parent Transactions without
+// touching the database
```

Comment:
Please, add details about the content since `HashMap<String, String>` is not descriptive enough.

Reply:
I was thinking to add type aliases
```
ReceiptId = String
TransactionHash = String
```
to be more explicit in the code. But I doubt it because we use Strings while real ids and hashes a `CryptoHash`. What do you think?

Reply:
Well, ReceiptId -> TransactionHash is an improvement, but it still does not communicate the fact that it is a relation for receipt to the transaction where that receipt is originated from.

Reply:
So don't you think such aliases might confuse more than help?